.github/workflows/mkdocs.yml

```diff
@@ -11,5 +13,10 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.x
-      - run: pip install mkdocs-material 
-      - run: mkdocs gh-deploy --force
+      - run: |
+          pip install mkdocs-material 
+          pip install mike
+      - run: | 
+        export REF=$(echo $GITHUB_REF | pcregrep --only-matching=1 ".*\/.*\/(.*)")
```

Comment:
This shell snippet works great locally (setting `GITHUB_REF` to stuff like `refs/heads/main` and `refs/tags/v1.2.3`). Need to be sure if this works in Actions though

Reply:
Tested here: https://github.com/jkayani/argocd-vault-plugin/runs/3561014531?check_suite_focus=true